packages/core/src/host/interface.ts

```diff
@@ -0,0 +1,161 @@
+/**
+ * Interface for the Alloy host file system operations.
+ *
+ * This interface abstracts file system operations to allow different
+ * implementations across different environments (Node.js, browser, etc.).
+ */
+export interface AlloyHostInterface {
```

Comment:
```suggestion
export interface AlloyHost
```

does it need the `Interface` suffix there?